proposals/init.md

```diff
@@ -0,0 +1,582 @@
+Init Only Members
+=====
+
+## Summary
+This proposal adds the concept of init only properties to C#. These properties
```

Comment:
```suggestion
This proposal adds the concept of init-only setters in properties and indexers to C#. These properties and indexers
```

Reply:
Ajred specifically went through and settled on `init only` throughout the document.

Reply:
Missed that. Seems wrong though, but the hyphen is a nit, so I'm fine with whatever Jared picks.

Reply:
> Not sure how to avoid repeating "properties or indexers" in the whole document.

I think it's okay to just refer properties in the document and call out that it also works for indexers in the details section.

Reply:
@jaredpar You may have missed some of my comments. The spec still doesn't mention indexers. I'll start a PR to correct some things.

---
In reply to: [411695374](https://github.com/dotnet/csharplang/pull/3367#discussion_r411695374) [](ancestors = 411695374)

Reply:
Didn't miss that comment. I added a mention of that + an explicit example using indexers to drive home it was intended to be supported. What I apparently missed was saving and pushing that to the doc 😦 

Saw you already updated the doc though to mention they're included so I think we're fine.